practice_06.15.2019_generics/Interfaces/IMyList.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace practice_06._15._2019_generics.Task1_MyList
+{
+    public interface IMyList<T>
+    {
+        void Add(T item);
```

Comment:
Please do not forget to separate members with an empty line.

Reply:
fixed